<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SearchView</class>
 <widget class="QWidget" name="SearchView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1057</width>
    <height>622</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="btnTrack">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>280</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Найти</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="trackPopular">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>130</y>
     <width>131</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Популярность</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="rok">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>200</y>
     <width>51</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>рок</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="rep">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>200</y>
     <width>51</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>реп</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="classic">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>200</y>
     <width>91</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>классика</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="indi">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>240</y>
     <width>61</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>инди</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="lofi">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>240</y>
     <width>61</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>lo-fi</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="pop">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>240</y>
     <width>61</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>поп</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>170</y>
     <width>101</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Жанры:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>100</y>
     <width>121</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Сортировать по:</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="trackDate">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>130</y>
     <width>131</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Дата выхода</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="numTrack">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>150</y>
     <width>131</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Кол-во песен</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="artistPopular">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>150</y>
     <width>131</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Популярность</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>110</y>
     <width>121</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Сортировать по:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="artistEntry">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>60</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="trackEntry">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>50</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>10</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>ТРЕКИ</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>20</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>ИСПОЛНИТЕЛИ</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>40</y>
     <width>81</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Поиск по
 названию:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>50</y>
     <width>81</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Поиск по
имени:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnArtist">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>280</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Найти</string>
   </property>
  </widget>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>330</y>
     <width>921</width>
     <height>221</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>250</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="sizeIncrement">
    <size>
     <width>50</width>
     <height>50</height>
    </size>
   </property>
   <property name="baseSize">
    <size>
     <width>50</width>
     <height>0</height>
    </size>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAsNeeded</enum>
   </property>
   <property name="sizeAdjustPolicy">
    <enum>QAbstractScrollArea::AdjustToContents</enum>
   </property>
   <attribute name="horizontalHeaderDefaultSectionSize">
    <number>130</number>
   </attribute>
   <attribute name="verticalHeaderCascadingSectionResizes">
    <bool>true</bool>
   </attribute>
  </widget>
  <widget class="QPushButton" name="btnMenu">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>500</y>
     <width>89</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;&lt;В МЕНЮ</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
